packages/main/src/plugin/kubernetes-port-forward-model.ts

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************
```

Comment:
Interfaces that will be shared between the main and renderer package should be moved to `packages/api/src`